kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -207,14 +208,34 @@ server {
 
             os.rename("/srv/storage/" .. tmp_file_name, "/srv/storage/" .. header_path_with_filename)
             os.remove("/srv/storage/" .. tmp_file_name .. ".shactx")
+
+            -- Send confirmation to the conductor after uploaded file.
+            local http = require "resty.http"
+            local httpc = http.new()
+            local res, err = httpc:request_uri("{{ upload_report_url }}" .. header_path_with_filename, {
+            method = "POST",
+            })
+
+            if (res.status < 200 or res.status >= 300) then
+                ngx.status = 502
+                error_response = '{"error_message":"Failed to report upload. Conductor responded with status {' .. res.status .. '}", "conductor_message":"' .. res.body .. '"}'
+                ngx.say(error_response)
+                -- Return response body in nginx log for debugging purposes
+                ngx.log(ngx.ERR, error_response)
+                return ngx.exit(502)
```

Comment:
Would it be possible to do `return ngx.exit(ngx.status)` here? You would not have to repeat the HTTP status twice.

Reply:
Well, I don't remember if I tested your solution, but everywhere in this file we use  convention `return ngx.exit(number)`

Reply:
Yes, I know. And it had never sit right with me because we're (seemingly unnecessarily) repeating the code twice. I did not see a better solution back then but now I have realized that we could just do what I proposed above. If that works then I think we should start doing it from now on.

Reply:
Done.